core/modules/replica/ControllerApp.cc

```diff
@@ -50,6 +55,98 @@ string const description =
     " operations in a replication setup.";
 
 using namespace lsst::qserv::replica;
+namespace util = lsst::qserv::util;
+
+void printAsTable(SqlResultSet const& result,
+                  bool sqlShowType) {
+
+    cout << "  error:     " << result.error << "\n"
+         << "  hasResult: " << (result.hasResult ? "yes" : "no") << "\n"
+         << "  #fields:   " << to_string(result.fields.size()) << "\n"
+         << "  #rows:     " << to_string(result.rows.size()) << "\n";
+
+    if (result.hasResult) {
+
+        size_t const numRows = result.rows.size();
+        size_t const numColumns = result.fields.size();
+
+        vector<shared_ptr<vector<string>>> tableDataByColumns;
+        tableDataByColumns.reserve(numColumns);
+
+        for (size_t columnIdx = 0; columnIdx < numColumns; ++columnIdx) {
+            auto tableColumnPtr = make_shared<vector<string>>();
+            tableColumnPtr->reserve(numRows);
+            tableDataByColumns.push_back(tableColumnPtr);
+        }
+        for (auto const& row :  result.rows) {                
+            for (size_t columnIdx = 0; columnIdx < numColumns; ++columnIdx) {
+
+                auto const& field  = result.fields[columnIdx];
+                auto const& cell   = row   .cells [columnIdx];
```

Comment:
I know you like to align things vertically but IMO the space in `row   .cells` goes too far. I'd probably accept
    auto const& field  = result.fields[columnIdx];
    auto const& cell   =     row.cells [columnIdx];

Reply:
Good suggestion! Honestly., I'm still undecided if I should keep aligning my code like I do it now, instead of just "streaming" it. I've doing it both. Apart from obvious benefits of the alignment, it also causes causes troubles when adding more lines. Then I may need to realign the rest of this "table".

Reply:
one of my objections to aligning is it can make for noisy code reviews when you have to realign a block.

Reply:
Yes, that's my concern too.